PWGLF/Tasks/QC/mcParticlePrediction.cxx

```diff
@@ -296,9 +342,9 @@ struct mcParticlePrediction {
     for (const auto& particle : mcParticles) {
       particle.pdgCode();
       const auto id = PIDExtended::pdgToId(particle);
-      // if (id < 0) { // always false
-      //   continue;
-      // }
+      if (id < 0) {
+        continue;
+      }
```

Comment:
This is reverting a fix made in https://github.com/AliceO2Group/O2Physics/pull/6479. Please revert the change.

Reply:
Fixed in https://github.com/AliceO2Group/O2Physics/pull/6760

Reply:
Thanks